rbd/data_pool_id.go

```diff
@@ -0,0 +1,22 @@
+//go:build ceph_preview
+
+package rbd
+
+// #cgo LDFLAGS: -lrbd
+// #include <stdlib.h>
+// #include <rados/librados.h>
```

Comment:
Remove these lines. They are unnecessary because you don't directly use any C functions from these headers.